### DISCUSSION TITLE
Error when trying to run executable in pixi environment: The file exists and is executable. Check the interpreter or linker?

### DISCUSSION THREAD

**Discussion by nwpulverizer** [prefix-dev/pixi#2041] [Q&A] [answered]
[pixi.toml.txt](https://github.com/user-attachments/files/16983407/pixi.toml.txt)
[pixi.lock.txt](https://github.com/user-attachments/files/16983408/pixi.lock.txt)
Hello I am new to using pixi but I really like it so far. I have run into an issue that I am not really sure how to solve. I installed a executable in a pixi environment (hexrdgui) but when I go to run it  - either just by typing the command `hexrdgui` or `pixi run hexrdgui` I cannot get it to run. 

The error if I run `pixi run hexrdgui` is 
```shell
Error launching 'hexrdgui': No such file or directory (os error 2)
```
and the error if I just run `hexrdgui` is
```shell
exec: Failed to execute process 'bla/bla/bla/.pixi/envs/default/bin/hexrdgui': The file exists and is executable. Check the interpreter or linker?
```

here is the pixi info output:

```shell
      Pixi version: 0.28.2
          Platform: linux-64
  Virtual packages: __unix=0=0
                  : __linux=6.10.9=0
                  : __glibc=2.40=0
                  : __cuda=12.6=0
                  : __archspec=1=skylake
         Cache dir: /home/nathan/.cache/rattler/cache
      Auth storage: /home/nathan/.rattler/credentials.json
  Config locations: No config files found

Project
------------
              Name: scripts
           Version: 0.1.0
     Manifest file: /home/nathan/Nextcloud/Documents/Mn metal data/scripts/pixi.toml
      Last updated: 12-09-2024 10:12:08

Environments
------------
       Environment: default
          Features: default
          Channels: conda-forge, hexrd
  Dependency count: 2
      Dependencies: hexrd, hexrdgui
  Target platforms: linux-64

```

I might be doing something completely wrong here so let me know if I am missing something obvious. 

Thanks, 

NP

**Comment by ruben-arts** [comment, marked as answer]
Hey @nwpulverizer I think we missed the notification. 

I'm not able to reproduce it as it just runs for me.:
```
~/hexrdgui 
❯ pixi i
✔ The default environment has been installed.

~/hexrdgui took 2s 
❯ pixi run hexrdgui
# STARTS CORRECTLY
~/hexrdgui took 17s 
❯ pixi shell 
 . "/tmp/pixi_env_X9z.sh"

~/hexrdgui 
❯  . "/tmp/pixi_env_X9z.sh"

~/hexrdgui via 🅒 scripts 
❯ hexrdgui
# STARTS CORRECTLY
```
Could you update to the latest pixi version. And try a `pixi clean` to avoid any weird issues that are already solved?

**Comment by nwpulverizer** [comment]
Sorry, this is my first time using pixi I should have thought of that though.
